YumemiInternTask/Contract/WeatherContract.swift

```diff
@@ -5,6 +5,10 @@ protocol WeatherView: AnyObject {
     /// 天気画像を表示する
     /// - parameter weather:Weather型の天気情報
     func showWeather(_ weather: Weather)
+
+    /// エラーを表示させる
+    /// - parameter message:エラーメッセージ
+    func showError(message: String)
```

Comment:
[API Design Guidelines](https://swift.org/documentation/api-design-guidelines/) に沿った命名を考えると楽かもしれません。

> Prefer method and function names that make use sites form grammatical English phrases.

例えば
```suggestion
    func showError(withMessage message: String)
```